elk/training/train.py

```diff
@@ -34,6 +35,15 @@ class Elicit(Run):
     cross-validation. Defaults to "single", which means to train a single classifier
     on the training data. "cv" means to use cross-validation."""
 
+    @staticmethod
+    def Default():
```

Comment:
nit: method names should always be lower case

Reply:
Fixed. Thanks